pages/qr.tsx

```diff
@@ -31,7 +30,12 @@ export default function QRCodePage() {
           undefined,
           videoEl.current
         )
-        window.location.href = appendUrlParams(result.getText())
+        const qrUrl = new URL(result.getText())
+        if (qrUrl.hostname === 'rcvr.app') {
+          window.location.href = appendUrlParams(qrUrl)
+        } else {
+          console.error('Found wrong url host', qrUrl.hostname)
```

Comment:
So for a user who doesn't see the console, we're failing silently?

I was against this from [start](https://github.com/railslove/recover-backlog/issues/120#issuecomment-760251270) and was later open to vetting, but failing completely is not ideal. What if I used rcvr.app/qr to scan a code from care.rcvr.app.
I suggest in this case, we simply show a warning message. No need for a fancy UI, just simple `confirm("Warning, this QR code is not part of RecoverApp, you may continue there, but any data you enter will be sent to ${qrUrl.hostname}")` would do.

Reply:
Okay, will do so.